lib/phoenix_inline_svg/helpers.ex

```diff
@@ -69,20 +67,15 @@ defmodule PhoenixInlineSvg.Helpers do
       svg_image("image_name", "collection_name", attrs)
 
   """
-  defmacro __using__([otp_app: app_name] = _opts) do
-    svgs_path = Application.app_dir(app_name,
-      PhoenixInlineSvg.Utils.config_or_default(:dir, "priv/static/svg/"))
+  defmacro __using__(_) do
+    svgs_path = PhoenixInlineSvg.Utils.config_or_default(:dir, "assets/static/svg/")
```

Comment:
There is no longer a need for this to be a variable it can just be piped directly into the next set

```suggestion
    PhoenixInlineSvg.Utils.config_or_default(:dir, "assets/static/svg/")
    |> find_collection_sets
    ...
```